src/main/java/net/pms/encoders/Player.java

```diff
@@ -409,8 +409,8 @@ public static void setSubtitleOutputParameters(String fileName, DLNAMediaInfo me
 		 */
 		if (!configuration.isDisableSubtitles() && params.sid == null && media != null) {
 			// Check for subtitles again
-			File video = new File(fileName);
-			FileUtil.isSubtitlesExists(video, media, false);
+//			File video = new File(fileName);
+//			FileUtil.isSubtitlesExists(video, media, false);
```

Comment:
@valib I don't think leaving commented out code is a good practice unless there's a good reason, the code ends up being even more unreadable. If there's a good reason, it should be explained in a comment IMO. @SubJunk has said the same before.

Reply:
@Nadahar I leave it here only for the case when testing will be wrong. It could be deleted when developers approve that thich change is working.